src/features/messaging/MessageStorage.ts

```
@@ -137,4 +137,50 @@ export class MessageStorage {
+      const allMessages: Message[] = [];
+      for (const line of lines) {
+        try {
+          const data: MessageStorageData = JSON.parse(line);
+          const message: Message = {
+            ...data,
+            roomName
+          };
+          allMessages.push(message);
+        } catch (parseError) {
+          // Skip malformed lines
+          continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)